scripts/build_simple_wheels.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
A simple shell script for generating wheels, avoiding the use of manylinux for now. The command `python3 -m build <directory>` is a replacement for the deprecated `python3 setup.py --bdist_wheel`. It will generate a `dist/` folder containing both a .tar file for the sdist, and a .whl file for the bdist. More options, including only building either sdist or bdist, can be found [here](https://github.com/pypa/build)